pilot/pkg/xds/cds.go

```diff
@@ -18,10 +18,13 @@ import (
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pkg/config"
 	"istio.io/istio/pkg/config/schema/gvk"
+
 )
 
+
 type CdsGenerator struct {
 	Server *DiscoveryServer
+	Delta  bool
```

Comment:
If we do GenerateDeltas, I think this is not needed?

Reply:
yes